benchmarks/benchmarks-core/build.gradle

```diff
@@ -8,6 +8,9 @@ dependencies {
     jmh 'io.dropwizard.metrics5:metrics-core:latest.release'
     jmh 'io.prometheus:simpleclient_common:latest.release'
 
+    jmh 'io.dropwizard.metrics:metrics-core'
+    jmh 'com.google.guava:guava'
```

Comment:
I'm guessing these were transitive deps before and with the `implementation` change it exposed that?

Reply:
@checketts Yes, that's the reason why I added them explicitly as they are not available with the changes any more.